src/assets/RustAsset.js

```diff
@@ -0,0 +1,93 @@
+const url = require('url');
+const path = require('path');
+const toml = require('toml');
+const {exec} = require('child-process-promise');
+
+const fs = require('../utils/fs');
+const JSAsset = require('./JSAsset');
+const localRequire = require('../utils/localRequire');
+
+const rustTarget = `wasm32-unknown-emscripten`;
+
+class RustAsset extends JSAsset {
```

Comment:
I think we should probably make a `WASMAsset` class to load precompiled `.wasm` files, and use it as a base class for this one. That way the runtime can be shared across multiple wasm languages.